static_eip/metadata.rb

```diff
@@ -9,3 +9,5 @@
 
 issues_url 'https://github.com/18F/identity-cookbooks/issues'
 source_url 'https://github.com/18F/identity-cookbooks'
+
+depends 'aws_metadata'
```

Comment:
I can get rid of this since I don't need our ruby libs to handle the metadata tokens